pystruct/learners/one_slack_ssvm.py

```diff
@@ -336,16 +340,15 @@ def _constraint_from_cache(self, X, Y, joint_feature_gt, constraints):
         return Y_hat, djoint_feature, loss_mean
 
     def _find_new_constraint(self, X, Y, joint_feature_gt, constraints, check=True):
-        if self.n_jobs != 1:
-            # do inference in parallel
+        if any([self.n_jobs == 1, self.pool == None]):
             verbose = max(0, self.verbose - 3)
             Y_hat = Parallel(n_jobs=self.n_jobs, verbose=verbose)(
                 delayed(loss_augmented_inference)(
                     self.model, x, y, self.w, relaxed=True)
                 for x, y in zip(X, Y))
         else:
-            Y_hat = self.model.batch_loss_augmented_inference(
```

Comment:
ok fixed this now, tests all pass. I'm working on the changes we've talked about in another branch, should have updates soon.